src/common/transformations/src/transformations/common_optimizations/convert_compression_only_to_legacy.cpp

```diff
@@ -32,6 +34,12 @@ bool ov::pass::ConvertCompressedOnlyToLegacy::run_on_model(const std::shared_ptr
     RUN_ON_MODEL_SCOPE(ConvertCompressedOnlyToLegacy);
     if (ngraph::op::util::has_decompression_converts(f)) {
         Manager manager(get_pass_config());
+        manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
+
```

Comment:
please add a comment - why do we set callback here?

Reply:
added comment